package.json

```diff
@@ -50,7 +50,9 @@
     "react": "16.6.3",
     "react-dom": "16.6.3",
     "react-router-dom": "4.3.1",
-    "sanitize.css": "8.0.0"
+    "react-toolbox": "^2.0.0-beta.13",
+    "sanitize.css": "8.0.0",
+    "src": "^1.1.2"
```

Comment:
- Remove src module, most likely parcel installed it when not finding a path.
- Make sure all packages have fixed versions